### DISCUSSION TITLE
Duda tiempo de ejecución parte 2

### DISCUSSION THREAD

**Discussion by jhonnylakes** [IIC2133-PUC/2024-1#192] [Tarea 3] [answered]
Hola. Tengo una pregunta con respecto a los tiempos pedidos de ejecución en la tarea en la parte 2. Ya logré que me corrieran todos los test case de la parte 2 con el output correcto, pero aún no he logrado hacer que el test-1 y el test-2 corran en menos de 10 segundos. No entiendo bien qué se espera que hagamos en el programa para que cumpla los requisitos de complejidad, yo implementé el algoritmo visto en clases de dar vuelto. Agradecería mucho una pequeña guía.

Por otro lado,  quería preguntar si van a considerar la poca anticipación con que fueron publicados los test case a la hora de corregir el tiempo de ejecución?? lo pregunto porque, además de mi caso, he sabido de otros compañeros que también tenían el programa de la tarea listo de hace más de una semana y dado lo tarde que fueron subidos los test case, están complicados con la entrega porque tuvieron que cambiar los algoritmos y el informe de las partes 2 y 3 ya que no sabían la escala de volumen de datos que se esperaba pudiera manejar el programa. Gracias de antemano.

**Comment by benjavicente** [comment, marked as answer]
Hola!

Vamos a compilar el código con un mayor nivel de optimización. Ahí, por lo que pude probar, tu código corre bien de tiempo. 

Voy a considerar el problema, pero de antemano creo que con el nivel de optimización no debería existir problemas en los test cases dado una implementación como la que tú realizaste.
